Prox/Prox/PlaceCarouselViewController.swift

```diff
@@ -144,7 +144,13 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
             // Make sure we only call this once, for testing purposes.
             if !once {
                 FirebasePlacesDatabase().getPlaces(forLocation: TEST_LL).upon(DispatchQueue.main) { places in
-                    self.placeCarousel.places = places.flatMap { $0 as? Venue }
+                    self.placeCarousel.places = places.flatMap { result in
+                        if result.isSuccess() {
```

Comment:
We could remove the complexity here if `PlacesResult.result` just returned an optional and we let flatMap take care of removing the bad entries.

But it works and it's a prototype so it's probably not worth changing now.